share/scripts/inverse/run_gromacs.sh

```diff
@@ -28,6 +28,27 @@ EOF
    exit 0
 fi
 
+tasks=$(get_number_tasks)
+
+mdrun_opts="$(csg_get_property --allow-empty cg.inverse.gromacs.mdrun.opts)"
+
+if [[ ${mdrun_opts} == *multidir* ]]; then
+
+  mdirs=$(echo ${mdrun_opts} | grep -o "sim" | wc -l)
```

Comment:
There is a bit of convention going on indeed, which must go into the manual eventually - the option has this structure: '-multidir sim_0 sim_1 sim_2 sim_3', i.e. the dir-names can be anything but should contain pattern 'sim' and end with sequential numbers starting with zero (the last requirement can be dropped, if really unwanted; upon some re-coding). 

All the input files must be provided with a suffix (ending) following the same convention: '.#' = dot followed by sequential numbers (starting with zero, not one). The master grompp which is used for the run collecting RDF stats, must have nsteps = mdirs * nsteps(subdir).